src/sql/plot.py

```diff
@@ -44,7 +46,14 @@ def _summary_stats(conn, table, column, with_=None):
 
     query = template.render(table=table, column=column)
 
-    values = conn.execute(query, with_).fetchone()
+    try:
+        values = conn.execute(query, with_).fetchone()
+    except ProgrammingError as e:
+        print(e)
+        raise exceptions.RuntimeError(
+            f"\nEnsure that percentile_disc " f"function is available on {driver}."
```

Comment:
Can we use one string instead of splitting it? e.g. `f"\nEnsure that percentile_disc function is available on {driver}."`

Reply:
This happens when I run `pkgmt format`